packages/sherlock/src/providers.js

```diff
@@ -258,15 +258,23 @@ class LocatorSuggestProvider extends (ProviderBase) {
   async search(searchString, maxresults) {
     const suggestUrl = `${this.url}/suggest?text=${searchString}&maxSuggestions=${maxresults}`;
 
-    const response = await fetch(suggestUrl);
-    const responseJson = await response.json();
-    const features = responseJson.suggestions.map(suggestion => {
-      return {
-        attributes: suggestion
-      };
-    });
-
-    return { data: features };
+    let response;
+    try {
+      response = await fetch(suggestUrl);
+      const responseJson = await response.json();
+      const features = responseJson.suggestions.map(suggestion => {
+        return {
+          attributes: suggestion
+        };
+      });
+
+      return { ok: true, data: features };
+    } catch {
+      const message = 'error with suggest request';
+      console.error(message , response);
+
+      throw new Error(message);
```

Comment:
**question**: should this return `ok: false` with the error message? or does the main code expect the error to be thrown? It seems like we have two competing patterns here

Reply:
The main code doesn't even check the `ok` prop. It expects an error to be thrown. Yes, I noticed that the `ok` stuff wasn't consistent. I can take another look and maybe just remove it and make sure that errors are thrown.

Reply:
OK. I gave it a try. Let me know what you think.